docs/developer/export-products.mdx

```diff
@@ -0,0 +1,355 @@
+---
+title: Exporting Products
+sidebar_label: Exporting products
+---
+
+import Chart from "@site/components/Chart";
+
+## Introduction
+This guide describes how to export products from the Saleor GraphQL API.
+Exporting products can be useful for creating backups of your data or for easy
+and quick bulk editing.
+
+You can export all products, filtered products, or products with specific IDs. 
+Products data can be exported to CSV or XLSX file, but CSV file is recommended because
+it is much less time-consuming to generate. You can also define which fields will be exported.
+If any product variants fields are specified then products and variants data are exported.
+Link to download file is sent by email to the requestor. If any error occurs then
+email with information about problems is sent.
+
+:::note
+When the export is made by the `App` email is not sent. You can get the file by query the
+`ExportFile` with ID returned from `exportProducts` mutation.
+:::
+
+### Export file structure
+Each row in the exported file represents a single product variant, but it also contains
+general product fields.
+For example, if a product has three variants, there will be three lines in total.
+Each line will contain common product fields such as name or description,
+and fields specific to given variant:
+
+| id | name         | description                    | variant sku | variant weight |
```

Comment:
Are the column titles actually lowercase?

Reply:
Yes